### ISSUE TITLE
Only variables should be passed by reference

### ISSUE THREAD

**Issue by mihdan** [Automattic/Edit-Flow#321]
`wp-admin/post.php?post=162634&action=edit`

```
Only variables should be passed by reference - /wp-content/plugins/edit-flow/modules/custom-status/custom-status.php:638 
#6 /wp-content/plugins/edit-flow/modules/custom-status/custom-status.php:422 - EF_Custom_Status->get_default_custom_status()
#5 [internal call] - EF_Custom_Status->post_admin_header('')
#4 /wp-includes/plugin.php:503 - call_user_func_array(Array[2], Array[1])
#3 /wp-admin/admin-header.php:120 - do_action('admin_print_sc...')
#2 /wp-admin/edit-form-advanced.php:413 - require_once('/var/www/roomb...')
#1 /wp-admin/post.php:218 - include('/var/www/roomb...')
```

**Comment by iandunn** [comment]
This is a duplicate of #311 

(The notices are slightly different, but essentially the same thing, and can all be fixed in a single issue)

**Comment by cojennin** [comment]
Closing, fixed by #340
